packages/frameworks/src/types.ts

```diff
@@ -1,3 +1,5 @@
+import { Route } from '@vercel/routing-utils/dist/types';
```

Comment:
```suggestion
import { Route } from '@vercel/routing-utils';
```

Reply:
I did it like this so that downstream users that only load the types (i.e. front only needs `Framework` type) don't have to load all of routing-utils into memory.

Reply:
Won't it be stripped at build time so it wont be emitted in the bundle js in front?

Also you could try: `import type { Route } from '@vercel/routing-utils';`

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-export

Reply:
Oh ya I think you're right, that's a good point.